src/test/java/com/redhat/red/build/finder/BuildFinderTest.java

```diff
@@ -48,7 +48,7 @@ public void verifyDebug() throws IOException {
 
             BuildFinder.main(new String[] {"-d", "-k", "-o", folder.getAbsolutePath(), target.getAbsolutePath()});
 
-            assertTrue(systemOutRule.getLog().contains("DEBUG"));
+            assertTrue(systemOutRule.getLog().contains(" DEBUG "));
```

Comment:
I don't think this commit is really needed? Its a string comparison so this seems redundant?

Reply:
Currently, the debug pattern is different from the default pattern from logback.xml. I wanted to be sure to match the debug pattern and not also the default pattern. In other words, the current test would pass even when the debug command-line option was not given.